packages/test/src/integration-tests/test-update.ts

```diff
@@ -0,0 +1,145 @@
+import * as wf from '@temporalio/workflow';
+import { helpers, makeTestFunction } from './helpers';
+import * as workflows from './workflows';
+
+const test = makeTestFunction({
+  workflowsPath: workflows.workflowsPath,
+  workflowEnvironmentOpts: {
```

Comment:
This should be optional.

Reply:
Yep -- as explained above, it is optional. But, for now, we need to set the server to `latest` for the Update tests.

Reply:
I'm adding a TODO to remind us to get rid of this `server.executable.version=latest` stuff but for now it seems appropriate to me; let me know if you'd prefer a different action here.